tools/sof_perf_analyzer.py

```diff
@@ -102,6 +100,8 @@ def collect_perf_info(trace_item: TraceItem):
     ppln_id = msg[0].split(':')[1]
     comp_id = int(msg[1], 16)
     perf_val = int(msg[5]), int(msg[7]), int(msg[10]), int(msg[12])
+    global AUDIO_PERIOD
+    AUDIO_PERIOD = int(msg[7])
```

Comment:
All other `msg[x]` fields go to either `CompPerfSample` or to `Component`. Why should `msg[7]` be special and require a `global` (and pylint warning)